README.md

```diff
@@ -1,2 +1,7 @@
-# conversation-model-to-sql-blueprint
-provides an example of how to design an SQL database for storing JSON data, specifically from the POST /api/v2/analytics/conversations/details/query endpoint. 
+# Conversation model to SQL Blueprint
+
```

Comment:
I see other blueprints link to the blueprint on the dev-centre from here. I guess I'll add that after this is added to the dev-centre?!

Reply:
You can add it now. The URL is based on the repo name, so https://developer.genesys.cloud/blueprints/conversation-model-to-sql-blueprint/